oozie-to-airflow/run-sys-test

```diff
@@ -61,12 +61,38 @@ export DATAPROC_USER=${DATAPROC_USER:=${USER:=example_user}}
 
 RUN_SSH_COMMAND="false"
 
+SETUP_CONFIGURATION="false"
+
 SETUP_AUTOCOMPLETE="false"
 
 COMPOSER_WEB_UI_URL=""
 
 VERBOSE="false"
 
+function setup_configuration {
+    read -r -p "Enter Composer name [${COMPOSER_NAME}]: " input
+    [[ -n "${input}" ]] && COMPOSER_NAME="${input}"
+    save_to_file COMPOSER_NAME
+
+    read -r -p "Enter Composer location [${COMPOSER_LOCATION}]: " input
+    [[ -n "${input}" ]] && COMPOSER_LOCATION="${input}"
+    save_to_file COMPOSER_LOCATION
+
+    read -r -p "Enter Dataproc cluster name [${DATAPROC_CLUSTER_NAME}]: " input
+    [[ -n "${input}" ]] && DATAPROC_CLUSTER_NAME="${input}"
+    save_to_file DATAPROC_CLUSTER_NAME
+
+    read -r -p "Enter GCP Region [${GCP_REGION}]: " input
+    [[ -n "${input}" ]] && GCP_REGION="${input}"
+    save_to_file GCP_REGION
+
+    "${MY_DIR}/confirm" "Do you want to configure additional options?"
+
+    read -r -p "Enter the name of the bucket that is used to store DAGs [${COMPOSER_DAG_BUCKET}]: " input
```

Comment:
I think we do not need that. If it is overridden with --bucket we should use it from there but we should refresh it every time we run fetch_composer_data, but otherwise we should fetch it from composer configuration.